File: Java/IntegerListFromUsp/src/Exercicio05.java
import java.util.Scanner;

public class Exercicio05 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        scanner.reset();
        System.out.println("Digite a o valores de venda do mês:");
        int[] vendas = new int[31];
        int maiorVenda = 0;
        int diaVenda = 0;
        for (int i = 0; i < 31; i++) {
            System.out.print("Vendas dia %d: ".formatted(i + 1));
            scanner.reset();
            vendas[i] = scanner.nextInt();
            if (vendas[i] > maiorVenda) {
                maiorVenda = vendas[i];
                diaVenda = i;
            }
        }

        System.out.println("Maior venda foi no dia %d no valor %d.".formatted(diaVenda + 1, maiorVenda));
    }
}

File: CPP/IntegerListFromUsp/src/Exercicio05.cpp
#include <iostream>

using namespace std;

int main() {
    const int dias = 31;
    int maiorVenda = -1;
    int diaMaior = 0;

    for (int dia = 1; dia <= dias; dia++) {
        int vendas;
        cout << "Vendas do dia " << dia << ": ";
        cin >> vendas;
        if (vendas > maiorVenda) {
            maiorVenda = vendas;
            diaMaior = dia;
        }
    }

    cout << "Maior venda no dia " << diaMaior << " com " << maiorVenda << " discos." << endl;
    return 0;
}
